LoopKitUI/Extensions/FloatingPoint.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FloatingPoint.swift
+//  LoopKitUI
+//
+//  Created by Michael Pangburn on 7/30/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+extension Double {
```

Comment:
Not generic over FloatingPoint since `pow` has no generic variant

Reply:
Strange, but ok... :)